src/components/common/Memorandum.vue

```diff
@@ -0,0 +1,343 @@
+<template>
+    <div id="upload-rules">
+
+      <v-card flat>
+
+        <!-- Header -->
+        <div class="section-container header-container">
+          <v-icon color="appDkBlue">mdi-format-list-text</v-icon>
+            <label class="font-weight-bold pl-2">Memorandum</label>
+        </div>
+
+        <div class="section-container">
+          <v-row no-gutters class="mt-4">
+            <v-col cols="2" class="pr-2">
+              <label>
+                <strong>Memorandum</strong>
+              </label>
+              <v-flex md1>
+                <v-chip
+                  v-if="isEditingMemorandum"
+                  id="corrected-lbl"
+                  x-small label
+                  color="primary"
+                  text-color="white"
+                >
+                  {{getEditedLabel}}
+                </v-chip>
+              </v-flex>
+            </v-col>
+
+            <!-- Display Mode -->
+            <template v-if="!isEditingMemorandum">
+              <v-col :cols="selectChange ? 10 : 9" class="pr-2">
+                <!-- Edit Mode -->
+                <!-- Instructional Text -->
+                <div v-if="!selectChange && getMemorandum?.length <= 0" class="mx-4">
```

Comment:
I don't think you can use `?` in the template? I could be wrong It's why the syntax loses all of it's color etc.